shared/js/background/https.es6.js

```diff
@@ -60,10 +63,28 @@ class HTTPS {
     }
 
     /**
-     * @param {string} host
+     * @param {string} url either domain (example.com) or a full URL (http://example.com/about)
      * @returns {Boolean|Promise<Boolean>} returns true if host can be upgraded, false if it shouldn't be upgraded and a promise if we don't know yet and we are checking against a remote service
      */
-    canUpgradeHost (host) {
+    canUpgradeUrl (url) {
```

Comment:
this function is only used in one place - this file (besides test), so I felt comfortable changing it